public/config.json

```diff
@@ -1,5 +1,14 @@
 {
-  "apiBase": "/client/api",
+  "servers": [
+    {
+      "name": "Local-Server",
+      "apiBase": "/client/api"
+    },
+    {
+      "name": "Primate-QA",
+      "apiBase": "http://primate-qa.cloudstack.cloud:8080/client/api"
```

Comment:
Should this be disclosed to public?

Reply:
@ravening In my opinion it's fine, but in your opinion is there a better way?

Reply:
@utchoang I meant about the URL.. isn't that owned by shapblue?

Reply:
@ravening This is example, I'll remove it later.